apps/site/assets/ts/leaflet/components/Map.tsx

```diff
@@ -74,8 +74,15 @@ const Component = ({
     const boundsOrByMarkers = bounds || (boundsByMarkers && getBounds(markers));
     const position = mapCenter(markers, defaultCenter);
     const nonNullZoom = zoom === null ? undefined : zoom;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const mapRef = React.createRef<any>();
+    const onTilesLoaded = (): void => {
+      mapRef.current.container.parentElement.classList += " map--loaded";
+    };
+
```

Comment:
I added a breakpoint right here (with a console.log statement, just so there was something to ground the breakpoint) and I seemed to see the "map-loaded" class getting added juuuuuust ahead of leaflet's actual load of the tiles, I think?  But the className doesn't get added when the polyline appears, or the markers, or the vehicle tooltips -- it consistently happens just *as* the tiles are being loaded (I think...).  So if that's enough for the test, then 👍 👍 

<img width="1439" alt="Screen Shot 2021-09-07 at 2 22 00 PM" src="https://user-images.githubusercontent.com/69368883/132393832-a0adf353-ba24-4b84-8e87-b171133aa7d1.png">

Reply:
you'd think a breakpoint there would mean that the whole map isn't even instantiated yet, right? 🤔 oh my gosh, that's super interesting!

Reply:
that said

> But the className doesn't get added when the polyline appears, or the markers, or the vehicle tooltips

that is expected, as the function is only on the `<TileLayer />`, which is where the map tiles (and nothing else) is 😅